spec/std/dir_spec.cr

```diff
@@ -257,6 +257,14 @@ describe "Dir" do
   it "lists entries" do
     filenames = Dir.entries(__DIR__)
     filenames.includes?("dir_spec.cr").should be_true
+    filenames.includes?(".").should be_true
+    filenames.includes?("..").should be_true
+  end
+
+  it "lists children" do
+    filenames = Dir.children(__DIR__)
+    filenames.includes?(".").should be_false
+    filenames.includes?("..").should be_false
```

Comment:
Example with `size - 2` knows about implementation details, and should be not used.